File: Core Java Eclipse IDE Workspace/AccessSpecifiersExample/src/bean/Employee.java
package bean;

public class Employee {
private int id;
String name;
protected float salary;
public String designation;

public void disEmp() {
	System.out.println("id "+id);
	System.out.println("name "+name);
	System.out.println("salary "+salary);
	System.out.println("Designation "+designation);
}
}

File: Phase 4/React JS Programs/welcome-app/src/Employee.js

import React from 'react';

class Employee extends React.Component {
    
    constructor(props){
        super(props);
        this.state = {id:100,name:"Ajay",age:21};
    }

    render() {
        this.setState({id:101,name:"Vijay",age:24});  
        return(
            <div>
                <h2>Employee component in class style</h2>
    <p>Id is {this.state.id} Name is {this.state.name} 
                Age is {this.state.age}</p>
                <p>Designation is {this.props.designation}</p>
            </div>
        )
    }
}

export default Employee;